buildutils/torque.spec.in

```diff
@@ -61,9 +61,9 @@
 %define ac_trqauthd_sock_dir %{?trqauthd_sock_dir:%{trqauthd_sock_dir}}%{!?trqauthd_sock_dir:/tmp}
 
 ### Build Requirements
-%define breq_cpuset   %{?with_cpuset:hwloc-devel}
+%define breq_cpuset   %{!?hwloc_path:%{?with_cpuset:hwloc-devel}}
 %define breq_gui      %{?with_gui:tcl-devel tk-devel tclx}
-%define breq_numa     %{?with_cpuset:hwloc-devel}
```

Comment:
These two changes look odd to me.  Why would having a path to hwloc mean that the TORQUE RPMs don't depend on the `hwloc-devel` RPM?  And why is `%{breq_numa}` referencing `%{with_cpuset}` anyway?  That shouldn't be the case, I don't think.